---a/net/minecraft/item/ItemEnderPearl.java
+++b/net/minecraft/item/ItemEnderPearl.java
@@ -6,29 +6,24 @@
 import net.minecraft.stats.StatList;
 import net.minecraft.world.World;
 
-public class ItemEnderPearl extends Item
-{
+public class ItemEnderPearl extends Item {
+
     private static final String __OBFID = "CL_00000027";
 
-    public ItemEnderPearl()
-    {
+    public ItemEnderPearl() {
         this.maxStackSize = 16;
         this.setCreativeTab(CreativeTabs.tabMisc);
     }
 
-    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn)
-    {
-        if (playerIn.capabilities.isCreativeMode)
-        {
+    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn) {
+        if (playerIn.capabilities.isCreativeMode) {
             return itemStackIn;
-        }
-        else
-        {
+        } else {
             --itemStackIn.stackSize;
             worldIn.playSoundAtEntity(playerIn, "random.bow", 0.5F, 0.4F / (itemRand.nextFloat() * 0.4F + 0.8F));
 
-            if (!worldIn.isRemote)
-            {
+            if (!worldIn.isRemote) {
                 worldIn.spawnEntityInWorld(new EntityEnderPearl(worldIn, playerIn));
             }
 
@@ -36,4 +36,4 @@
             return itemStackIn;
         }
     }
-}
+}
